src/ScanItems.js

```diff
@@ -224,17 +233,21 @@ class ScanItems extends React.Component {
   }
 
   checkout = (barcode) => {
-    const { mutator: { checkout } } = this.props;
+    const { mutator: { checkout }, patronBlockOverriddenInfo } = this.props;
```

Comment:
Could you please split props destructuring to several line?

Reply:
Done. But I really don't like this way to steal the file space with replacing props. As for me would be better to replace on the new line destructuring if there are more than 3 properties only